http.ts

```diff
@@ -33,8 +33,8 @@ interface ServeEnv {
 // TODO: make them async function after this change is done
 // https://github.com/tc39/ecma262/pull/1250
 // See https://v8.dev/blog/fast-async
-export function serveConn(env: ServeEnv, conn: Conn) {
-  readRequest(conn).then(maybeHandleReq.bind(null, env, conn));
+function serveConn(env: ServeEnv, conn: Conn, bufr?: BufReader) {
```

Comment:
Please add to the above docs about what happens when bufr is/isn't provided. It's not clear to me from the signature. (And please reformat the above comment into jsdoc pls)

Reply:
Added